src/one_dragon_qt/widgets/draggable_list.py

```
@@ -0,0 +1,700 @@
+        self._items.insert(position, item)
+        self._layout.insertWidget(position, item)
+
+        # 更新后续项的索引
+        self._update_indices()
+
+    def remove_item(self, position: int) -> None:
+        """
+        移除指定位置的列表项
+
+        Args:
+            position: 要移除的位置（索引）
```

Ruff (0.14.10)

Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)